test/integration/pairs_test.lua

```diff
@@ -58,19 +63,41 @@ g.before_each(function()
             end
         ]])
     end
-end)
+end
+
+g_memtx.before_all = function() before_all(g_memtx, 'memtx') end
+g_vinyl.before_all = function() before_all(g_vinyl, 'vinyl') end
+
+g_memtx.after_all = function() after_all(g_memtx) end
+g_vinyl.after_all = function() after_all(g_vinyl) end
+
+g_memtx.before_each(function() before_each(g_memtx) end)
+g_vinyl.before_each(function() before_each(g_vinyl) end)
+
+local function add(name, fn)
+    g_memtx[name] = fn
+    g_vinyl[name] = fn
+end
+
+local function unflatten(tuple, metadata)
+    local object = {}
+    for fieldno, field in ipairs(metadata) do
+        object[field.name] = tuple[fieldno]
+    end
+    return object
+end
 
-local function insert_customers(customers)
+local function insert_customers(g, customers)
     local inserted_objects = {}
 
     for _, customer in ipairs(customers) do
-        local obj, err = g.cluster.main_server.net_box:eval([[
+        local result, err = g.cluster.main_server.net_box:eval([[
             local crud = require('crud')
             return crud.insert('customers', ...)
         ]],{customer})
 
         t.assert_equals(err, nil)
-
+        local obj = unflatten(result.rows[1], result.metadata)
```

Comment:
Why not `crud.unflatten_rows`?

Reply:
Thanks, changed to `crud.unflatten_rows`